.github/ISSUE_TEMPLATE/bug.yml

```
@@ -45,7 +45,7 @@ body:
     id: network
     attributes:
       description: Which network(s) did the issue occur on?
-      label: Hedera network
+      label: Hiero network
```

```suggestion
      label: Hedera network
```

I think this stays as hedera network for main net test net and preview net. I think more brand neutrality changes need to happen first

---

Thank you.. let me revert this one